thirdeye/thirdeye-frontend/package.json

```diff
@@ -88,7 +87,7 @@
     "toastr": "^2.1.4"
   },
   "engines": {
-    "node": "10.*"
+    "node": ">=10.*"
```

Comment:
When merging this commit, please keep changes of package.json from this PR: https://github.com/apache/incubator-pinot/pull/5418/files

Reply:
We have to include this line, and otherwise it will not pass the ELR check.

Reply:
I updated the Node part.  Please keep the rest of the PR when merging.